src/components/header_menu.jsx

```diff
@@ -1,11 +1,81 @@
-import React from 'react'
-
-const HeaderMenu = props => (
-  <header className={'page-header-menu ' + (props.className || '')}>
-    <div className="page-header-menu--left">{props.left}</div>
-    {props.children}
-    <div className="page-header-menu--right">{props.right}</div>
-  </header>
-)
+import React, { useMemo } from 'react'
+
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Link from 'cozy-ui/transpiled/react/Link'
+import { SharedRecipients } from 'cozy-sharing'
+import { Typography } from '@material-ui/core'
```

Comment:
You have to use Typography from cozy-ui, not from mui.

Reply:
Will do